sdks/python/build.gradle

```diff
@@ -204,14 +204,10 @@ task directRunnerIT(dependsOn: 'installGcpTest') {
 
 // Before running this, you need to:
 //
-// 1. build the SDK container:
```

Comment:
Perhaps we should keep the alternative documented here, since running against an existing job server is useful, for example when working on/debugging the runner. In the side input PR I already made some changes to the options here, to utilize Gradle properties.

Reply:
+1